launch/tier4_control_launch/config/trajectory_follower/lateral_controller.param.yaml

```diff
@@ -46,7 +46,7 @@
     input_delay: 0.24                # steering input delay time for delay compensation
     vehicle_model_steer_tau: 0.3     # steering dynamics time constant (1d approximation) [s]
     steer_lim_deg: 40.0              # steering angle limit [deg]
-    steer_rate_lim_dps: 600.0        # steering angle rate limit [deg/s]
+    steer_rate_lim_dps: 20.0         # steering angle rate limit [deg/s]
```

Comment:
Are there any reasons for changing this value? (600.0 seems too much, but is 20.0 reasonable value?)

Reply:
Yes 600.0 was too much, This value depends your vehicle interface, and max turning velocity of your steering wheel. We are currently testing this PR on Isuzu tests, and low level controller limited steering rate to 20.0 degree. So, it is just vehicle specific parameter.

Reply:
@mitsudome-r 

Now it is 40.0 considering the sample_vehicle. It seems more realistic than 600 dps where the vehicle can do the impossible :) But since it vehicle spec. param it should be modified for every vehicle.